pickATrip_django_apps/templates/pickATrip_django_apps/base.html

```diff
@@ -41,14 +41,15 @@
         </li>
 		{% if user.is_authenticated %}
 		<li class="nav-item">
-          <a class="nav-link " aria-current="page" href={% url 'profile' %}>profile</a>
+          <a class="nav-link " aria-current="page" href={% url 'createPost' %}>Create new post</a>
         </li>
 		<li class="nav-item">
-          <a class="nav-link " aria-current="page" href={% url 'logout' %}>Logout</a>
+          <a class="nav-link " aria-current="page" href={% url 'profile' %}>profile</a>
```

Comment:
you can capitalize the 'profile' to 'Profile':
` <a class="nav-link " aria-current="page" href={% url 'profile' %}>Profile</a>`